tests/network-bridge-firewall

```diff
@@ -117,6 +117,71 @@ firewallTests() {
     ! lxc exec c1 -- ping -c1 2001:db8::1 || false
 
     lxc delete -f c1
+
+    # Perform NAT tests for managed bridges only.
+    if [ $managed -eq 1 ]; then
```

Comment:
we're going to want a condition for testing this only on lxd 5.21 onwards as i dont intend to backport the fix back to 5.0

Reply:
Fixed. Now the only failing tests are the ones for `5.21/edge` as it's not yet backported.